theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -126,40 +127,35 @@ def test_exp_over_1_plus_exp(self):
 
             # tests inv_1_plus_exp
             f = theano.function([x], T.fill(x, 1.0) / (1 + T.exp(-x)), mode=m)
-            assert hasattr(f.maker.fgraph.outputs[0].tag, 'trace')
+            assert check_stack_trace(f, ops_to_check=sigmoid)
```

Comment:
The stack trace is missing from the optimization [`local_exp_over_1_plus_exp`](https://github.com/lamblin/Theano/blob/fix_stack_check_nnet/theano/tensor/nnet/sigm.py#L578-L623), which is why the test is failing.